app/controllers/user_controller.rb

```diff
@@ -126,6 +126,7 @@ def account
          (params[:user][:auth_provider] == current_user.auth_provider &&
           params[:user][:auth_uid] == current_user.auth_uid)
         update_user(current_user, params)
+        @title = t "user.account.title"
```

Comment:
Would it not be more sensible just to move setting of the title to later in the method rather than duplicating it here?

Reply:
It makes much more sense - I didn't realise that the `redirect_to` method didn't immediately exit the method